__tests__/items-count.js

```diff
@@ -0,0 +1,27 @@
+import { JSDOM } from 'jsdom';
+
+const document = new JSDOM().window.document;
+
+const itemsContainer = document.createElement('div');
+document.body.appendChild(itemsContainer);
+
+// mock get items
+const getItems = () => Array(24).fill(0);
+
+const displayItems = () => {
```

Comment:
- [ ] Currently, your tests are failing. So kindly fix that as well :wink: 

![image](https://github.com/AlyAtalla/js-capstone2/assets/102699958/789503e5-5542-4d77-bf93-5b9235d94622)

Reply:
Thank you.